app/serializers/api/v2/serializable_provider.rb

```diff
@@ -1,10 +1,21 @@
 module API
   module V2
     class SerializableProvider < JSONAPI::Serializable::Resource
+      class << self
+        def enrichment_attribute(name, enrichment_name = name)
+          attribute name do
+            @object.enrichments.last&.__send__(enrichment_name)
```

Comment:
just for me https://apidock.com/ruby/Object/__send__

but can we write like
@object.enrichments.last&.send enrichment_name

the `__send__` makes me think of js